src/main/java/com/rarchives/ripme/ripper/rippers/MyhentaicomicsRipper.java

```diff
@@ -122,53 +123,134 @@ public Document getNextPage(Document doc) throws IOException {
             return albumPagesList;
         }
 
+    public List<String> getAlbumsFromPage(String url) {
+        List<String> pagesToRip;
+        List<String> result = new ArrayList<String>();
+        logger.info("Running getAlbumsFromPage");
+        Document doc;
+        try {
+            doc = Http.url("http://myhentaicomics.com" + url).get();
+        } catch(IOException e){
+            logger.warn("Failed to log link in Jsoup");
+            doc = null;
+            e.printStackTrace();
+        }
+        // This for goes over every album on the page
+        for (Element elem : doc.select("li.g-album > a")) {
+            String link = elem.attr("href");
+            logger.info("Grabbing album " + link);
+            pagesToRip = getNextAlbumPage(link);
+            logger.info(pagesToRip);
+            for (String element : pagesToRip) {
+                Document album_doc;
+                try {
+                    logger.info("grabbing " + element + " with jsoup");
+                    boolean startsWithhttp = element.startsWith("http://");
+                    if (startsWithhttp == false) {
+                        album_doc = Http.url("http://myhentaicomics.com/" + element).get();
+                    }
+                    else {
+                        album_doc = Http.url(element).get();
+                    }
+                } catch(IOException e){
+                    logger.warn("Failed to log link in Jsoup");
+                    album_doc = null;
+                    e.printStackTrace();
+                }
+                for (Element el :album_doc.select("img")) {
+                    String imageSource = el.attr("src");
+                    // This bool is here so we don't try and download the site logo
+                    boolean b = imageSource.startsWith("http://");
+                    if (b == false) {
```

Comment:
!b

But honestly I don't think you need the boolean var if you're only going to use it in this one test. You can just inline it.